ibis/bigquery/udf/api.py

```diff
@@ -12,6 +13,13 @@
 
 __all__ = ('udf',)
 
+# NOTE: used for skip doctest for Python 3.8
```

Comment:
do we have an issue for this? 

define these constants in ibis/compat.py instead